web/olga/charts/static/charts/graphs.js

```diff
@@ -119,8 +119,51 @@
         Plotly.plot(chart, data, layout, {displayModeBar: false});
     }
 
+    function appendSecChart(chart, chart_data, chart_title) {
+        let time = Object.keys(newData.monthly);
+        let studNum = [];
+        let certNum = [];
+        let enthNum = [];
+        for (let item in newData.monthly) {
+            studNum.push(newData.monthly[item][0]);
+            certNum.push(newData.monthly[item][1]);
+            enthNum.push(newData.monthly[item][2]);
+        }
+
+
```

Comment:
Too many empty lines, please leave not more than two.

Reply:
done